src/games/strategy/triplea/ai/proAI/util/ProMoveUtils.java

```diff
@@ -271,6 +272,39 @@ public static void calculateBombardMoveRoutes(final PlayerID player, final List<
     }
   }
 
+  public static void calculateBombingRoutes(final PlayerID player, final List<Collection<Unit>> moveUnits,
```

Comment:
So is moveUnits. It changes 2 variables so would need to create a new class in order to return them both.